app/components/authorShowHeader/index.tsx

```diff
@@ -56,21 +56,21 @@ class AuthorShowHeader extends React.PureComponent<AuthorShowHeaderProps, Author
                 <div className={styles.metricInformation}>
                   {(author.paperCount || author.paperCount === 0) && (
                     <div className={styles.metricWrapper}>
-                      <span className={styles.metricValue}>{author.paperCount}</span>
+                      <span className={styles.metricValue}>{author.paperCount.toLocaleString()}</span>
                       <span className={styles.metricLabel}>Publications</span>
                     </div>
                   )}
 
                   {(author.hIndex || author.hIndex === 0) && (
                     <div className={styles.metricWrapper}>
-                      <span className={styles.metricValue}>{author.hIndex}</span>
+                      <span className={styles.metricValue}>{author.hIndex.toLocaleString()}</span>
                       <span className={styles.metricLabel}>H-index</span>
                     </div>
                   )}
 
                   {(author.citationCount || author.hIndex === 0) && (
                     <div className={styles.metricWrapper}>
-                      <span className={styles.metricValue}>{author.citationCount}</span>
+                      <span className={styles.metricValue}>{author.citationCount.toLocaleString()}</span>
```

Comment:
For the above three changes, I wonder why did you use `toLocaleString` method. `toLocaleString` method aims `Date` type. if you want to change them to String type object, you can use `toString()` method.

Reply:
I've got it why you tried to use `toLocaleString` method.
You can use `formatNumber` helper method in `app/helpers/formatNumber.tsx`